opensrp-app/src/main/java/org/smartregister/util/LangUtils.java

```diff
@@ -36,13 +36,14 @@ public static Configuration setAppLocale(Context context, String language) {
 
```

Comment:
@yussin-hidayat  I think we should replace line 35 with `Locale.forLanguageTag(language).` This should add support for tag values that follow a well-formed **BCP 47** standard